README.md

```diff
@@ -2,8 +2,20 @@
 
 This repository is used for generalized Express.js discussions.
 
-* [Current discussions](https://github.com/expressjs/discussions/issues?q=is%3Aissue+is%3Aopen+label%3Adiscuss)
-* [Upcoming meetings](https://github.com/expressjs/discussions/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20label%3Ameeting)
+- [Current discussions](https://github.com/expressjs/discussions/issues?q=is%3Aissue+is%3Aopen+label%3Adiscuss)
+- [Upcoming meetings](https://github.com/expressjs/discussions/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aopen%20label%3Ameeting)
+
+## Current v5 Meeting Cadence
+
+While focusing on releasing Express v5, the meeting schedule has been changed from the normal schedule listed below. Until further notice, meetings will be held on this cadence.
```

Comment:
This (and the other lines) should be hard wrapped like the rest of the doc :)

Reply:
Updated wrapping to 80 chars